sql/parser/sql.y

```diff
@@ -54,8 +55,8 @@ package parser
   limit          *Limit
   targetList     TargetList
   targetListPtr  *TargetList
-  privilegeType  PrivilegeType
-  privilegeList  PrivilegeList
+  privilegeType  structured.Privilege
```

Comment:
me too, but having structured depend on parser wasn't much better.
we have a other options:
* move privilege proto to its own thing (possible under parser). this would have the nice property of variables being privilege.ALL etc..
* have constants defined in the parser package, although this makes structured depend on parser again
* have constants elsewhere, but sql/parser will still have to depend on it